src/Datadog.Trace/Configuration/TracerSettings.cs

```diff
@@ -296,6 +308,18 @@ public bool DiagnosticSourceEnabled
         /// </summary>
         public bool StartupDiagnosticLogEnabled { get; set; }
 
+        /// <summary>
+        /// Gets or sets the HTTP status code that should be marked as errors for server integrations.
+        /// </summary>
+        /// <seealso cref="ConfigurationKeys.HttpServerErrorStatusCodes"/>
+        public bool[] HttpServerErrorStatusCodes { get; set; }
+
+        /// <summary>
+        /// Gets or sets the HTTP status code that should be marked as errors for client integrations.
+        /// </summary>
+        /// <seealso cref="ConfigurationKeys.HttpClientErrorStatusCodes"/>
+        public bool[] HttpClientErrorStatusCodes { get; set; }
```

Comment:
These are perhaps worth a comment explaining the implementation.
i.e., "There is an entry per status code and the index of the status code has a bool indicating whether or not it is an error"